### DISCUSSION TITLE
Is it possible to skip the "Verify your account" prompt before signing?

### DISCUSSION THREAD

**Discussion by mrkkrly** [rainbow-me/rainbowkit#2186] [Q&A] [answered]
I would like to achieve the following flow: when a user clicks on 'Connect Wallet', the wallet is connected, and immediately afterward, the user is prompted to sign a message with their wallet. I came across a discussion about this a year ago, has any solution been found since then?

**Comment by DanielSinclair** [comment, marked as answer]
We support this with our Authentication API to prompt the user with a Sign-in with Ethereum message upon connect. When signing, the user is authorized against your app's backend. Reference our Next.js example [here](https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-next-siwe-next-auth), and the docs [here](https://www.rainbowkit.com/docs/authentication). let me know if you have any questions!
